### ISSUE TITLE
question: why add_edge(object, subject, pred) ?

### ISSUE THREAD

**Issue by saramsey** [biolink/ontobio#315]
I notice that in the `ontobio` source code (e.g., in `obograph_util.py` line 62 and in ontol.py line 153), the convention in the underlying `networkx.DiGraph` data model *appears to be* that an arc goes from object to subject, and not (as I would expect if reading a triple aloud) subject to object. Do I understand this correctly?

**Comment by cmungall** [comment]
nx doesn't have a specific subject/object terminology, but we can assume that the nx add_edge method takes an object-subject pair:

```
>>> import networkx as nx
>>> G=nx.DiGraph()
>>> G.add_edge(1,2)
>>> G.add_edge(2,3)
>>> nx.ancestors(G,1)
set()
>>> nx.ancestors(G,3)
{1, 2}
```

Since ancestor is normally taken to be subject->object

It's annoying how every community has it's own terminology and conventions...

@dustine32 you recently encountered this with obonet should we add something to the documentation?

**Comment by dustine32** [comment]
@cmungall Yeah we discussed this a bit in issue #179 and I really liked your [comment](https://github.com/biolink/ontobio/issues/179#issuecomment-393733239) explaining the justification for the ontobio object-subject reversal.

Perhaps we can add this same comment to the readthedocs under [here](https://ontobio.readthedocs.io/en/latest/concepts.html#identifiers)?
